src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -673,6 +673,7 @@ describe('ReactUpdates', () => {
   it('should flush updates in the correct order across roots', () => {
     var instances = [];
     var updates = [];
+    var container = document.createElement('div');
```

Comment:
Hm. This looks wrong. This changes the behavior we're testing. Why doesn't it work?